packages/airnode-validator/test/validator.test.ts

```diff
@@ -1,10 +1,19 @@
 import * as fs from 'fs';
 import path from 'path';
-import { validateWithTemplate } from '../src';
+import dotenv from 'dotenv';
+import template from 'lodash/template';
+import { validateWithTemplate, validateJsonWithTemplate } from '../src';
 
 const tests = fs.readdirSync(path.resolve(__dirname, 'validatorTests'));
 const validOutput = { valid: true, messages: [] };
 
+// Regular expression that does not match anything, ensuring no escaping or interpolation happens
+// https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L199
+const NO_MATCH_REGEXP = /($^)/;
+// Regular expression matching ES template literal delimiter (${}) with escaping
+// https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L175
+const ES_MATCH_REGEXP = /\$\{([^\\}]*(?:\\.[^\\}]*)*)\}/g;
+
```

Comment:
Copy & paste from the `airnode-node` package is really not an ideal solution because if the implementation from the package changes the tests will still pass.

I would suggest refactoring a bit (if you must) the `parseConfig` function and importing the result instead.

Reply:
I can simply use the `parseConfig` function from `airnode-node`, but current jest config is causing trouble importing it to tests in `airnode-validator`, im not sure how can I import it, perhaps the config might have to change...

Reply:
I'm ok with that if it's needed.

Reply:
There is a TODO in node and we can refactor it when the times comes. I'd like this to happen after 0.4